test/Microsoft.DotNet.Tools.Tests.Utilities/FileExtensions.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.Tools.Test.Utilities
+{
+    public static class FileExtensions
+    {
+        public static string Executable
+        {
+            get
+            {
+#if NET451
```

Comment:
You could use platformabstractions like we do in `Utils` to avoid the need for an ifdef here.